sdk/Lusid.Sdk.Tests/Utilities/InstrumentExamples.cs

```diff
@@ -155,6 +155,93 @@ internal static LusidInstrument CreateExampleSwap()
             );
         }
 
+        internal static InterestRateSwaption CreateExampleInterestRateSwaption()
+        {
+                        // CREATE an Interest Rate Swap (IRS)
+            var startDate = new DateTimeOffset(2020, 2, 7, 0, 0, 0, TimeSpan.Zero);
+            var maturityDate = new DateTimeOffset(2030, 2, 7, 0, 0, 0, TimeSpan.Zero);
+
+            // CREATE the flow conventions, index convention for swap
+            var flowConventions = new FlowConventions(
+                scope: null,
```

Comment:
if these are not required they should be defaulted to null rather than force the user to specify null